include/boost/process/v2/impl/pid.ipp

```diff
@@ -22,6 +66,215 @@ pid_type current_pid() {return ::GetCurrentProcessId();}
 pid_type current_pid() {return ::getpid();}
 #endif
 
+#if defined(BOOST_PROCESS_V2_WINDOWS)
+
+std::vector<pid_type> all_pids(error_code & ec)
```

Comment:
I wonder if that should be able to take a custom allocator.

Reply:
You mean using malloc or new instead of std::vector?

Reply:
no, something like `std::pmr::polymorphic_allocator`. But I am unsure that's worth it here.

Reply:
let me know once you've made up your mind on this

Reply:
yeah it's nonsense. we're iterating through the process table, a few allocs won't matter.